ipysurfer/stc.py

```diff
@@ -28,9 +28,11 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
     hemi : str, 'lh' | 'rh' | 'split' | 'both'
         The hemisphere to display.
     colormap : str | np.ndarray of float, shape(n_colors, 3 | 4)
-        Name of colormap to use or a custom look up table. If array, must
+        Only str type is supported.
```

Comment:
I'm a bit confused here - so you cannot input an array? If so, then we should just remove the `np.ndarray ...etc` from the first line of this argument. Or, clarify by saying "only str type is currently supported" to imply that it'll be supported in the future